src/TestFramework/DataFaker.php

```diff
@@ -1007,6 +1007,16 @@ public function soapId()
     }
 
     /**
+     * Return a refund reason
+     *
+     * @return string
+     */
+    public function refundReason()
+    {
+        return $this->note();
```

Comment:
@parth21shah since it's in the data faker, i just added a new function for refundReason and left `note` since something else could have a note maybe. If you prefer I could mark the note function as deprecated.